addons/stock/models/stock_move.py

```diff
@@ -193,6 +194,11 @@ def _default_group_id(self):
     show_quant = fields.Boolean("Show Quant", compute="_compute_show_info")
     show_lots_m2o = fields.Boolean("Show lot_id", compute="_compute_show_info")
     show_lots_text = fields.Boolean("Show lot_name", compute="_compute_show_info")
+    product_custom_attribute_value_ids = fields.One2many(
+        comodel_name='product.attribute.custom.value', inverse_name='res_id',
```

Comment:
missing a domain on the `res_model`, same for the other models with custom values